### PR TITLE
[v3.1.x] Review Documentation of WSO2 API Management Pattern Helm Resources

### PR DESCRIPTION
**Pull request by chirangaalwis**

### CODE REVIEW

File: advanced/am-pattern-1/README.md [hunk 1]
```diff
@@ -138,6 +153,7 @@ b. Add the above hosts as entries in `/etc/hosts` file as follows:
 c. Try navigating to `https://<RELEASE_NAME>-am/carbon`, `https://<RELEASE_NAME>-am/publisher` and `https://<RELEASE_NAME>-am/devportal` from your favorite browser.
 
 ## Configuration
+
 The following tables lists the configurable parameters of the chart and their default values.
```
Comment (chamindi-a) [inline advanced/am-pattern-1/README.md:158]: list?

File: advanced/am-pattern-2/README.md [hunk 2]
```diff
@@ -39,16 +43,19 @@
...
 
 [Option 2] Deploy WSO2  API Manager using Docker images from WSO2 Private Docker Registry.
+
 ```
 helm install --name <RELEASE_NAME> wso2/am-pattern-2 --version 3.1.0-2 --namespace <NAMESPACE> --set wso2.subscription.username=<SUBSCRIPTION_USERNAME> --set wso2.subscription.password=<SUBSCRIPTION_PASSWORD>
 ```
+
 **Note:**
 
 * `NAMESPACE` should be the Kubernetes Namespace in which the resources are deployed.
```
Comment (chamindi-a) [inline advanced/am-pattern-2/README.md:61]: if we can take the bullet and bring the line after the ':' better
there is another instance of the same few lines down

File: advanced/am-pattern-2/README.md [hunk 3]
```diff
@@ -39,16 +43,19 @@
 ### Install Chart From [WSO2 Helm Chart Repository](https://hub.helm.sh/charts/wso2)
 
 ##### 1. Deploy Helm chart for WSO2 APIM Pattern 2 deployment
+
 [Option 1] Deploy using Docker images from DockerHub.
```
Comment (chamindi-a) [inline advanced/am-pattern-2/README.md:47]: these options can we tab it and give a different number for all the places you have [option]

File: advanced/am-pattern-1/README.md (hunk 1 above)
Comment (chirangaalwis) [reply advanced/am-pattern-1/README.md:158]: @chamindi-a how about `defines`?

Comment (chamindi-a) [reply advanced/am-pattern-1/README.md:158]: but since you are listing 
lets keep the 'list'

**Comment by chirangaalwis** [comment]
@chamindi-a I have updated the outdated external links with Kubernetes API version `1.14` [documentation](https://v1-14.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/). This is since we have tested out the deployments with this version of Kubernetes.

Further, these user input options may change in the future and we will take steps to update them accordingly.

**Comment by chirangaalwis** [comment]
@chamindi-a I will update this issue after [this](https://github.com/wso2/kubernetes-apim/milestone/14) milestone has been covered, as other tasks may lead to changes in the README documentation.

File: advanced/am-pattern-2/README.md (hunk 3 above)
Comment (chamindi-a) [reply advanced/am-pattern-2/README.md:47]: We can make this issue as Resolved

File: advanced/am-pattern-2/README.md (hunk 2 above)
Comment (chamindi-a) [reply advanced/am-pattern-2/README.md:61]: Can make the status to Resolve

**Comment by chamindi-a** [comment]
> @chamindi-a I have updated the outdated external links with Kubernetes API version `1.14` [documentation](https://v1-14.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/). This is since we have tested out the deployments with this version of Kubernetes.
> 
> Further, these user input options may change in the future and we will take steps to update them accordingly.

Thank you Chiranga to the update on this, I have checked the external links and all seem fix
Resolved mentioned issues above

> @chamindi-a I have updated the outdated external links with Kubernetes API version `1.14` [documentation](https://v1-14.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/). This is since we have tested out the deployments with this version of Kubernetes.
> 
> Further, these user input options may change in the future and we will take steps to update them accordingly.

Thank you Chiranga for updating this PR, I have checked the external links and all seem fix
Resolved mentioned issues above

**Comment by chirangaalwis** [comment]
Change to the PR title as this covers all three patterns.

**Pull request merged**